sections/security/hideerrors.md

```diff
@@ -0,0 +1,34 @@
+# Hide error details from client
+
+### One Paragraph Explainer
+Exposing application error details to the client in production should be avoided due to risk of exposing sensitive application details such as server filepaths, third party modules in use, and other internal workings of the application which could be exploited by an attacker.
+Express comes with a built-in error handler, which takes care of any errors that might be encountered in the app. This default error-handling middleware function is added at the end of the middleware function stack.
+If you pass an error to `next()` and you do not handle it in a custom error handler, it will be handled by the built-in Express error handler; the error will be written to the client with the stack trace. This behaviour will be true when `NODE_ENV` is set to `development`, however when `NODE_ENV` is set to `production`, the stack trace is not written, only the HTTP response code. 
+
+### Code example: Express error handler
+``` javascript
```

Comment:
@js-kyle consider making it a short example, 4-5 lines of code (users just need to get the overall idea)